src/vorta/store/settings.py

```diff
@@ -48,6 +48,17 @@ def get_misc_settings() -> List[Dict[str, str]]:
             'group': startup,
             'label': trans_late('settings', 'Open main window on startup'),
         },
+        {
+            'key': 'run_in_background',
+            'value': False,
+            'type': 'checkbox',
+            'group': startup,
+            'label': trans_late(
+                'settings',
+                'Run vorta in background',
+            ),
```

Comment:
How does this new setting work differently from `enable_background_question`? Can't you adjust that setting to work as needed?

Reply:
Hi! I'm a bit confused here! I actually was unable to see the checkbox for the `enable_background_question` in mics tab, that's why I added a different one 😓, but now it's visible for me!

Reply:
>  I actually was unable to see the checkbox

I noticed that too. Settings without a group aren't displayed at the moment (which is a bug). But you can easily work around that by adding the group attribute.

> but now it's visible for me

That's weird.

Reply:
> > I actually was unable to see the checkbox
> 
> I noticed that too. Settings without a group aren't displayed at the moment (which is a bug). But you can easily work around that by adding the group attribute.
> 
> > but now it's visible for me
> 
> That's weird.

Hmm did and it displayed the setting, but if I restore the setting to previous state i.e without the group attribute it still displays the setting checkbox and it works as expected without any changes

Reply:
![weird](https://user-images.githubusercontent.com/34976093/217886509-5ff810fe-1419-4f87-ac58-11e6f9348400.gif)

Here's, the example


Code from `settings.py`

```
settings += [
            {
                'key': 'enable_background_question',
                'value': True,
                'type': 'checkbox',
                'label': trans_late(
                    'settings',
                    "Toggle run in background dialogue",
                ),
            },
            {
                'key': 'disable_background_state',
                'value': False,
                'type': 'internal',
                'label': 'Previous background exit button state',
            },
        ]
```
back to original version

Reply:
Ah, This is because you local settings database still has the group stored inside it. Vorta doesn't remove old settings keys.

Reply:
> Ah, This is because you local settings database still has the group stored inside it. Vorta doesn't remove old settings keys.

um, is there any way to remove it ? 😅

Reply:
You can edit the database located in `~/.local/share/Vorta/settings.db`. I use [sqlitebrowser](https://sqlitebrowser.org/) for that.

Reply:
> You can edit the database located in `~/.local/share/Vorta/settings.db`. I use [sqlitebrowser](https://sqlitebrowser.org/) for that.

Thank You 😄